CHANGELOG.md

```diff
@@ -7,10 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [1.0.4] - 2018-6-6
+## [1.1.0] - 2018-06-06
```

Comment:
You don't need this line. The `releasy` script already add version in CHANGELOG

Reply:
I know, but I thought it'd be better to edit it manually while the date is not being properly formatted, in order to avoid inconsistencies between releases.

Reply:
So, you can't use releasy. If you used, it will be 
`## [1.1.0] - 2018-6-6`
`## [1.1.0] - 2018-06-06`